src/handlers/comment/handlers/assign.ts

```diff
@@ -109,14 +112,17 @@ export const assign = async (body: string) => {
     await addAssignees(issue.number, [payload.sender.login]);
   }
 
+  const days = Math.floor((new Date().getTime() - new Date(issue.created_at).getTime()) / (1000 * 60 * 60 * 24));
+  const isBountyStale = staleBounty == 0 ? null : staleBounty > days ? false : true;
```

Comment:
`staleBounty > days`

[Here](https://github.com/ubiquity/ubiquibot/blob/7fd7d3b83aaab0b19673d9c601937d9fdeb3f18c/src/bindings/config.ts#L80) `ms(staleBountyTime)` returns milliseconds so it seems that we are comparing amount of milliseconds with the amount of days